src/i18n/en.json

```diff
@@ -1071,7 +1071,8 @@
     "filter.pill.owner": "Owner",
     "filter.pill.custom": "Custom",
     "expenditure.draft": "Review",
-    "expenditure.cancelled": "Failed",
+    "expenditure.cancelled": "Canceled",
```

Comment:
This once spelled canceled, and another time cancelled (double l's)

AFAIK they're both correct, just that one is the US and the other the British way of spelling it.

We just decide which one, and update this accordingly

Reply:
We should stick with single "l".